CryptoAnalysis/src/main/java/crypto/HeadlessCryptoScanner.java

```diff
@@ -104,6 +110,22 @@ public static HeadlessCryptoScanner createFromOptions(String... args) {
 		} else {
 			callGraphAlogrithm = CG.CHA;
 		}
+		if(options.hasOption("ignorePackagesFilePath")) {
```

Comment:
please rename option into ignorePackages.

Reply:
Also, please extract the option name into a constant *somewhere*. Is there some constants class or one where all the option names are stored, @AnakinSklavenwalker?

Reply:
unfortunately not, but i'd like to see one. Maybe not in this PR though.

Reply:
Renamed option to ignorePackages